cfpb/consumerfinance.gov
# Mega-menu: Remove unused event handler check

## Removals

- Remove unneeded mega-menu check for the parent of the current menu when clicking the page body.

## How to test this PR

1. Pull branch and run `gulp clean && gulp build`
2. Open the mega-menu and click between menus and outside of the menu.
3. Menu should close when clicked outside of the menu and remain open when clicking inside the menu.
